ui/oce/filters/inputs/range.jsx

```diff
@@ -16,6 +17,21 @@ let labelFormatter =  number => {
 }
 
 class Range extends translatable(Component){
+  maybeGetInputRange(min, max, minValue, maxValue, onUpdate){
```

Comment:
maybeGetInputRange can be renamed as getInputRange and the the IE condition can be checked in the render function, or change the name for something like getInputRangeIfNoIE